birdhouse/config/proxy/default.env

```diff
@@ -1,7 +1,6 @@
 # All env in this default.env can be overridden by env.local.
 
-# Folder inside "proxy" container to drop extra monitoring config
-export CANARIE_MONITORING_EXTRA_CONF_DIR="/conf.d"
+export PROXY_IMAGE="nginx"
```

Comment:
I would rather pin the specific latest version at this point in time.  Because `latest` version is not reproducible.  It changes on each build.  

Reproducibility is very important if we want to be able to diagnose problem across multiple different deployments of PAVICS.

For a given PAVICS tag, we should know precisely which version of the binary is being deployed.  We do allow image override via `env.local` but that's mostly for testing or the staging env.  For production deployment, I think and hope most users will want to stick to the tested versions coming from the stack.

Reply:
Sure and I agree. This is literally just a copy from https://github.com/Ouranosinc/CanarieAPI/blob/master/Dockerfile#L1 to maintain the existing settings but I'll explicitly pin it to whatever is the latest going forward.